junit-jupiter-params/src/main/java/org/junit/jupiter/params/converter/DefaultArgumentConverter.java

```diff
@@ -246,6 +248,15 @@ public Object convert(String source, Class<?> targetType) throws Exception {
 			return CONVERTERS.get(targetType).apply(source);
 		}
 
+		private static Class toClass(String type) {
```

Comment:
`Class<?>`

Reply:
Oops. :blush:

Reply:
That's the reason for the failing build. Any other changes needed?

Reply:
No, none so far.